src/i_joystick.c

```diff
@@ -153,6 +398,11 @@ void I_InitJoystick(void)
         return;
     }
 
+    if (use_gamepad)
+    {
+        return I_InitGamepad();
```

Comment:
Should this proceed with the old joystick code if it fails? `I_InitGamepad` needs to return bool in this case and fall through here.

Reply:
I say no. The SDL_GameController interface is built on top of SDL_Joystick, so I'm guessing there would be a failure with both interfaces. Furthermore, if `use_gamepad` is set then that means the controller binds are specific to SDL_GameController. Buttons and axes will get scrambled if those binds are applied to SDL_Joystick.